pytest_client_tools/plugin.py

```diff
@@ -91,15 +92,32 @@ def candlepin(request):
             "--candlepin-container-is-running"
         )
         initial_wait = 0
+        verify = False
         if start_container:
-            stack.enter_context(_create_candlepin_container())
+            container = _create_candlepin_container()
+            stack.enter_context(container)
             # candlepin takes a while to start
             initial_wait = 5
+            temp_cert = tempfile.NamedTemporaryFile()
+            stack.enter_context(temp_cert)
+            subprocess.run(
+                [
+                    "podman",
+                    "cp",
+                    f"{container.running_id}:/etc/candlepin/certs/candlepin-ca.crt",
+                    temp_cert.name,
+                ],
+                check=True,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+            )
```

Comment:
hmm considering there is already a class representing a podman container, so running a "podman cp" command manually feels a bit like going around it :)

I just pushed a new simple helper called `Podman.cp()`: 06e0546aae2c43ef0d82d89e1e0543fd278ab793
you can use it like this:
```python
container.cp("$C:/path/in/container", "/path/outside/container")
```
the literal `$C` string represents the ID of the container, and will be replaced dynamically

Reply:
nice thats handy!